build_aarch64_wheel.py

```diff
@@ -300,7 +305,9 @@ def build_torchvision(host: RemoteHost, *,
     if host.using_docker():
         build_vars += " CMAKE_SHARED_LINKER_FLAGS=-Wl,-z,max-page-size=0x10000"
 
-    host.run_cmd(f"cd vision; {build_vars} python3 setup.py bdist_wheel")
+    # LD_PRELOAD the libtorch_cpu.so to work around the 'cannot allocate memory in static TLS block' issue
+    host.run_cmd(f"export LD_PRELOAD=$HOME/miniforge3/lib/python{python_version}/site-packages/torch/lib/libtorch_cpu.so; cd vision; {build_vars} python3 setup.py bdist_wheel")
```

Comment:
This is not a valid workaround, we need to figure out what is causing it

Reply:
I did some more digging today and looks like it's OpenBLAS source built libraries consuming more TLS memory. Not sure why it wasn't observed earlier.